### ISSUE TITLE
emoji_support contains bad content where there is a literal string

### ISSUE THREAD

**Issue by longkai** [google/yamlfmt#62]
So when you set `emoji_support` to true, adding a test case to the `unicode_test.go`:

```go
		{
			name: "literal string",
			yamlStr: `a: |
  hello 😄\n`,
			expectedStr: `a: |
  hello 😄\n`,
		},
```

Which results:

```
parsed string does not match:
        expected: a: |
          hello 😄\n
        got: a: "hello 😄n"
```

I suppose the literal string turns one line due to the `go-yaml` library. However, the tail `n` is unacceptable.

**Comment by braydonk** [comment]
I notice you've closed it now, but here's what I've got about the issues with the output just to keep track.

Of the 3 issues with the output:
* Double quotes around the result
  - This is from the yaml library. I haven't yet thought of a context free way to get rid of these due to the way they are added into the formatted output, but I'm experimenting with solutions.
* Outputting to a single line
  - This is also from the library, and this one I'm not sure why it does that. It's possible the `yaml.Node` serialization doesn't retain the `|` multiline literal syntax information.
* The trailing `n`
  - This was a bug with `emoji_support` in 0.5.0 that I fixed in #55 and will be in version 0.6.0

**Comment by longkai** [comment]
Haha I also found the issue it should advanced one byte to check 'U'.

Besides I figure it out why it always format to a single since it consider emoji is [**not** printable](https://github.com/go-yaml/yaml/blob/f6f7691b1fdeb513f56608cd2c32c51f8194bf51/emitterc.go#L1332), so the text contains special characters and always output to escaped one line.
